addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1309,15 +1311,12 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
      * @param {Element} el
      */
     _scrollWebsiteAnimate(el) {
-        const scroll = $(el).scrollTop();
-        // Handle reverse scrolling
-        const direction = (scroll < this.lastScroll) ? -1 : 1;
-        this.lastScroll = scroll;
-
```

Comment:
This code is useless. Actually `direction` was always `1` and never `-1` before this PR.
But that changed when I set the "trottle" interval of `__onScrollWebsiteAnimate` to `10` instead of `200`. With this change, the direction does change from `1` to -`1` when scrolling up, but it introduces a bug, the animations are launched several times at the wrong time (it's hard to explain but by testing you can easily see the bug)